examples/shared/nrf24l01p/README.md

```diff
@@ -0,0 +1,21 @@
+This is a simple test/example for nRF24L01+ with
+STM32F429Disco. Two nRF24L01+ should be connected
+to the following board's pins:
+  TX:
```

Comment:
This Markdown file is badly formatted. I suggest list items (or table) to make it readable

Reply:
Done

Reply:
For NRF24L01 Air_Data_Rate should be Rate_1Mbps.